src/environment/types.ts

```diff
@@ -0,0 +1,47 @@
+import { TEnvironment } from "@zero-tech/zdc";
+
+
+export interface IZNSEnvironment {
```

Comment:
Although it is an object and ordering doesn't matter for functions that use this, it will still be easier to read if all the nullable properties come after non-nullable ones as opposed to an arbitrary order